README.md

```diff
@@ -1,55 +1,69 @@
-# BAMM Aspect Meta Model ![build](https://github.com/OpenManufacturingPlatform/sds-bamm-aspect-meta-model/actions/workflows/antora-build.yml/badge.svg)
+# Semantic Aspect Meta Model (SAMM) ![build](https://github.com/eclipse-esmf/esmf-semantic-aspect-meta-model/actions/workflows/pull-request-check.yml/badge.svg)
 
 - [Introduction](#introduction)
 - [Example Usage](#example-usage)
 - [Getting help](#getting-help)
 - [Build and contribute](#build-and-contribute)
 
 ## Introduction
-The [BAMM Aspect Meta Model](https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/snapshot/index.html) (BAMM) allows the creation of models to describe the semantics of digital twins by defining their domain specific aspects.
-In this context, digital twins are the digital representation of a physical or virtual object that bundles and combines several aspects.
-The BAMM Aspect *Meta* Model (BAMM) provides a set of predefined objects (as depicted below) that allow a domain expert to define aspect models and complement a digital twin with a semantic foundation.
-In case you were wondering what the B in BAMM refers to: BAMM is recursively defined and thus the B refers to BAMM itself.
+The Semantic Aspect Meta Model (SAMM) allows the creation of models to describe the semantics of
+digital twins by defining their domain specific aspects. In this context, digital twins are the
+digital representation of a physical or virtual object that bundles and combines several aspects.
+SAMM provides a set of predefined objects (as depicted below) that allow a domain expert to define
+aspect models and complement a digital twin with a semantic foundation.
 
-This repository contains the detailed documentation of the BAMM specification as an AsciiDoc documentation.
-The source files (AsciiDoc) are built using [Gradle](https://gradle.org/) and [Antora](https://antora.org/), which generates the documentation as HTML files.
+SAMM was previously known as BAMM (BAMM Aspect Meta Model).
 
-![BAMM Aspect Meta Model (BAMM) Elements](src/docs/modules/ROOT/images/aspect-meta-model.svg)
+This repository contains the detailed documentation of the SAMM specification as an
+[Antora](https://antora.org/) module , as well the formal specification parts as
+[SHACL](https://www.w3.org/TR/shacl/) shapes.
+
+The source files (AsciiDoc) are built using [Gradle](https://gradle.org/) and
```

Comment:
Are we not using Maven to build the doc now?

Reply:
Of course yes, this is an oversight :)